gordo/machine/dataset/data_provider/ncs_reader.py

```diff
@@ -20,11 +20,15 @@ class NcsReader(GordoBaseDataProvider):
     ASSET_TO_PATH = {
         "1101-sfb": "/raw/corporate/IMS Statfjord/sensordata/1101-SFB",
         "1110-gfa": "/raw/corporate/Aspen MS - IP21 Gullfaks A/sensordata/1110-GFA",
+        "1111-gfb": "/raw/corporate/Aspen MS - IP21 Gullfaks B/sensordata/1111-GFB",
+        "1112-gfc": "/raw/corporate/Aspen MS - IP21 Gullfaks C/sensordata/1112-GFC",
         "1125-kvb": "/raw/corporate/PI System Operation Norway/sensordata/1125-KVB",
         "1130-troa": "/raw/corporate/Aspen MS - IP21 Troll A/sensordata/1130-TROA",
         "1138-val": "/raw/corporate/PI System Operation Norway/sensordata/1138-VAL",
         "1163-gdr": "/raw/corporate/PI System Manager Sleipner/sensordata/1163-GDR",
         "1180-nor": "/raw/corporate/PI System Operation North/sensordata/1180-NOR",
+        "1175-kri": "/raw/corporate/Aspen MS - IP21 Kristin/sensordata/1175-KRI",
```

Comment:
This path seems to be incorrect. 
Also, would like if the next line was changed so it is in sorted order on the number.

Reply:
Nice catch. I was sure that you were wrong and double checked and... dangit! :)

Reply:
Haha, you can't trick a trickster :smile: 
Otherwise it looks good :tada: